mk-hackage.nix

```diff
@@ -85,11 +85,10 @@ let
         ${pkgs.haskell-nix.nix-tools.${compiler-nix-name}}/bin/hackage-to-nix $out 01-index.tar "https://mkHackageNix/"
       '';
 
-    copySrc = src: pkgs.runCommand "copied-src-${builtins.baseNameOf src}" {
-      __contentAddressed = true;
-    } ''
-      cp -T -r ${src} $out
-    '';
+    copySrc = src: builtins.path {
+      path = src;
+      name = "copied-src-${builtins.baseNameOf (builtins.unsafeDiscardStringContext src)}";
```

Comment:
A bit puzzled on why `unsafeDiscardStringContext` is needed but passing a string with context as `name` was causing an errors like
```
the string 'copied-src-dxk8mk38rkq57c8qfcan4vg16c2jb6i4-source' is not allowed to refer to a store path (such as '')
```

Reply:
maybe ask? try the other builtins?

Reply:
I'm asking you :)
do you mean other builtins instead of `path`?

Reply:
yes

Reply:
mmm tbh `builtins.path` seems to me the right choice here, let discuss this on slack